### ISSUE TITLE
the following arguments are required: emulator

### ISSUE THREAD

**Issue by Lacomon** [JartanFTW/kot-skipper#26]
1. Go to 'kot-skipper'
2. Click on 'main.py'
4. See error "the following arguments are required: emulator"

Expected behavior:
I installed everything that was in the requirements, but the program does not see the emulator

Screenshots:
https://user-images.githubusercontent.com/111542848/185511600-fcfe8b8a-a66c-437c-8732-d65cedec26a0.png

Desktop:
 - OS: Windows
 - Emulator: bluestacks
 - Version: 5.9.0.1061 n32

Additional context:
got this error, although the emulator was open

**Comment by JartanFTW** [comment]
Please see README.md:

> Example use: `main.py bluestacks -gd 50000 --gems 4 6 r5 b`

The program was made with supporting other emulators in mind, so you need to specify bluestacks in your command as seen above.

**Comment by Lacomon** [comment]
I don't quite understand what it's about, where do I need to enter the command?
I'm sorry if I'm asking stupid questions, I just recently discovered python.

**Comment by JartanFTW** [comment]
You'll have to google how to use command line arguments.
This might help: https://docs.microsoft.com/en-us/windows/terminal/command-line-arguments?tabs=windows

**Comment by Lacomon** [comment]
I still have the same error.
Can you recommend me the version of bluestacks that you used to work on?

**Comment by ZetaReserX** [comment]
tell me where to type this command

**Comment by Lacomon** [comment]
I found where to write this command
Everything works
"main.py bluestacks --gems b7 b8 g7 g8 p7 p8 r7 r8 y7 y8"
Is this command correct to search only for gold gems?

**Comment by JartanFTW** [comment]
> Is this command correct to search only for gold gems?

Yes, what you have will work. It can also be written as: "main.py bluestacks --gems 7 8"

To be honest I'm not sure the easiest way to explain to new users where to run the command in the simplest way possible. Since you just figured it out, how would you explain it?

**Comment by Lacomon** [comment]
I would explain everything step by step
Install everything that is in the "requirements" section (you can even leave comments or links)
Open bluestacks and the game, open cmd as administrator
Write "cd" and specify the path to the bot, for example "cd C:\Users\User\Downloads\kot-skipper-main
 And write the command "main.py bluestacks --gems"
and explain how to combine the arguments
To make it much easier, you can record a video
